03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>I think this would be more readable with one output per code block. Rather than print(property), just have each code block call <code>action.property</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>